coralproject/cay
# Max min

## What does this PR do?

Allows you to set Min. Chars/ Max. Chars in your textQuestions
## How do I test this PR?

Create Form and select Short Answer or Long Answer.
Try to set max and min values for chars.
Try to use invalid chars and find edge cases.

( To try the coundown too, checkout the max-min branch in elkhorn)

Thanks!

( Note: I didn't want to make long PR, but we should take that logic even further. And we should try to avoid extending classes )

@coralproject/frontend
